sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -281,10 +281,10 @@
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.getConfigurationSetting#string-string-OffsetDateTime}
      *
      * @param key The key of the setting to retrieve.
-     * @param label The label of the configuration setting to retrieve, or optionally, null if a setting with
-     * label is desired.
-     * @param acceptDateTime To access a past state of the configuration setting, or optionally, null if a setting with
-     * {@code acceptDateTime} is desired.
+     * @param label The label of the configuration setting to retrieve, or optionally, null if the label is not desired
+     * in the request.
+     * @param acceptDateTime To access a past state of the configuration setting, or optionally, null if the
```

Comment:
Similar to the `label` Javadocs, let's use the positive case.

```suggestion
     * @param acceptDateTime To access a past state of the configuration setting. If {@code null} then the current state of the configuration setting will be returned.
```